packages/plugin-expo-app/test/app.test.js

```diff
@@ -12,35 +12,7 @@ describe('plugin: expo app', () => {
     jest.resetModules()
   })
 
-  it('should should use revisionId if defined (all platforms)', done => {
-    const VERSION = '1.0.0'
-    const REVISION_ID = '1.0.0-r132432'
-
-    jest.doMock('expo-application', () => ({}))
-    jest.doMock('expo-constants', () => ({
-      default: {
-        platform: {},
-        manifest: { version: VERSION, revisionId: REVISION_ID }
-      }
-    }))
-
-    const plugin = require('..')
-
-    const c = new Client({ apiKey: 'api_key', plugins: [plugin] })
-
-    c._setDelivery(client => ({
-      sendEvent: (payload) => {
-        const r = JSON.parse(JSON.stringify(payload))
-        expect(r).toBeTruthy()
-        expect(r.events[0].app.codeBundleId).toBe(REVISION_ID)
-        done()
-      },
-      sendSession: () => {}
-    }))
-    c.notify(new Error('flip'))
-  })
-
-  it('should record nativeVersionCode on android', done => {
+  it('should record nativeVersionCode and versionCode on android', done => {
```

Comment:
Should these be split out into separate tests?

Reply:
Seems fine to be in one test, especially with the amount of setup/mocking